spinalcordtoolbox/reports/qc.py

```diff
@@ -561,7 +561,12 @@ def layout(qslice):
     try:
         from sys import platform as _platform
         if _platform == "linux" or _platform == "linux2":
-            sct.printv('xdg-open "{}/index.html"'.format(path_qc), type='info')
+            try:  
+                os.environ["BROWSER"]
+                sct.printv('xdg-open "{}/index.html"'.format(path_qc), type='info')
+            except KeyError: 
```

Comment:
could you please comment in the code when this KeyError is supposed to happen?

Reply:
also, are you 100% sure that in ALL linux platforms, this BROWSER variable exists if a browser is accessible?

Reply:
Xdg-open was asking for it, but now I am not sure that it is automatically set. 
The error is thrown if the variable is not previously declared or empty. 
I see two option : 
* there is webbrowser python package that can do the test without going through the $BROWSER variable.
* We can set a $DOCKER env variable for the docker container and use the same Key error.

Reply:
> We can set a $DOCKER env variable for the docker container and use the same Key error.

I like this approach.

Reply:
it seems a bit cleaner somehow...